src/test/kotlin/dev/stalla/model/itunes/ItunesCategoryTest.kt

```diff
@@ -127,127 +129,5 @@ internal class ItunesCategoryTest {
-            "News",
-            "Religion & Spirituality",
-            "Science",
-            "Society & Culture",
-            "Sports",
-            "Technology",
-            "True Crime",
-            "TV & Film"
-        )
-
-        @JvmStatic
-        private val nestedCategoryNames = listOf(
-            "Books",
-            "Design",
-            "Fashion & Beauty",
-            "Food",
-            "Performing Arts",
-            "Visual Arts",
-            "Careers",
-            "Entrepreneurship",
-            "Investing",
-            "Management",
-            "Marketing",
-            "Non-Profit",
-            "Comedy Interviews",
-            "Improv",
-            "Stand-Up",
-            "Courses",
-            "How To",
-            "Language Learning",
-            "Self-Improvement",
-            "Comedy Fiction",
-            "Drama",
-            "Science Fiction",
-            "Alternative Health",
-            "Fitness",
-            "Medicine",
-            "Mental Health",
-            "Nutrition",
-            "Sexuality",
-            "Education for Kids",
-            "Parenting",
-            "Pets & Animals",
-            "Stories for Kids",
-            "Animation & Manga",
-            "Automotive",
-            "Aviation",
-            "Crafts",
-            "Games",
-            "Hobbies",
-            "Home & Garden",
-            "Video Games",
-            "Music Commentary",
-            "Music History",
-            "Music Interviews",
-            "Business News",
-            "Daily News",
-            "Entertainment News",
-            "News Commentary",
-            "Politics",
-            "Sports News",
-            "Tech News",
-            "Buddhism",
-            "Christianity",
-            "Hinduism",
-            "Islam",
-            "Judaism",
-            "Religion",
-            "Spirituality",
-            "Astronomy",
-            "Chemistry",
-            "Earth Sciences",
-            "Life Sciences",
-            "Mathematics",
-            "Natural Sciences",
-            "Nature",
-            "Physics",
-            "Social Sciences",
-            "Documentary",
-            "Personal Journals",
-            "Philosophy",
-            "Places & Travel",
-            "Relationships",
-            "Baseball",
-            "Basketball",
-            "Cricket",
-            "Fantasy Sports",
-            "Football",
-            "Golf",
-            "Hockey",
-            "Rugby",
-            "Running",
-            "Soccer",
-            "Swimming",
-            "Tennis",
-            "Volleyball",
-            "Wilderness",
-            "Wrestling",
-            "After Shows",
-            "Film History",
-            "Film Interviews",
-            "Film Reviews",
-            "TV Reviews"
-        )
-
-        @JvmStatic
-        private val allItunesCategoryNames = simpleCategoryNames + nestedCategoryNames
-    }
+    companion object // this context for reflection
```

Comment:
Not sure what this means?

Reply:
Oh boy, this is just written proof that I had to code too much JavaScript the last couple of years. 

So to get the properties of the `ItunesCategory` companion, we need to call the getter (`member.getter.call(this)`). The `call` method needs at least one argument, and we provide `this`, which resolves in this context (pun intended) to the companion of the test class. Hence we need to declare a companion, and the comment is just there to remind us why we have an empty companion in the first place.

You are totally right of course, this comment needs improvement :)

Reply:
Now I'm just wondering...the `this` is there because the code is copied directly from the implementation which uses reflection as well (there `this` is the companion of `ItunesFactory`). In the test class, it's of course not, but the companion of the test class. So why does this work at all... 🤔

Reply:
> I had to code too much JavaScript the last couple of years.

Condolences 🤣 

As for why this works... sorry, too brain fried right now to figure it out :/

Reply:
Same here. Hopefully after a good nights sleep.

Reply:
One can hope!